### PR TITLE
Update Samba HOWTO with wsdd and IPv6 sections

### PR DESCRIPTION
**Pull request by ermo**
## Description

- Add section on Web Service Discovery Daemon (wsdd)
- Add section on IPv6 configuration with the default Solus Samba config

### Submitter Checklist

- [x] Updated the "lastmod" portion at the top of any modified Markdown files.
- [x] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

File: software/samba/en.md [hunk 1]
```diff
@@ -35,15 +36,84 @@ sudo systemctl enable --now smb
...
+sudo systemctl restart smb
+
 # Stop Samba manually
 sudo systemctl stop smb
 
 # Configure Samba to not start automatically on each boot and immediately stop the service
 sudo systemctl disable --now smb
 ```
 
+### How to start the Web Service Discovery Daemon to enable Windows 7+ Network discovery
```
Comment (JoshStrobl) [inline software/samba/en.md]: This is a really long header. Maybe something simpler like "Enabling Windows 7+ Network Discovery" then have a short explanation about wsdd?

File: software/samba/en.md [hunk 2]
```diff
@@ -35,15 +36,84 @@ sudo systemctl enable --now smb
...
+# Stop wsdd manually
+sudo systemctl stop wsdd
+
+# Configure wsdd to not start automatically on each boot and immediately stop the service
+sudo systemctl disable --now wsdd
+```
+
 For more details on managing services on Solus with *systemctl*, see `man systemctl` which is part of the systemd system and service manager.
 
+### How to access a Solus Samba instance via IPv6
```
Comment (JoshStrobl) [inline software/samba/en.md]: Maybe "Accessing Solus Samba via IPv6"

Comment (ermo) [reply software/samba/en.md]: Agreed.

File: software/samba/en.md (hunk 1 above)
Comment (ermo) [reply software/samba/en.md]: Good idea. Will fix.

**Pull request merged**